applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -180,7 +184,7 @@ def Initialize(self):
     def InitializeSolutionStep(self):
         if self._TimeBufferIsInitialized():
             # Recompute the BDF2 coefficients
-            (self.bdf_process).Execute()
+            (self.time_discretization).ComputeBDFCoefficients(self.GetComputingModelPart())
```

Comment:
@rubenzorrilla I would like to clarify:
The idea of the lightweight bdf-classes was that you don't have to do this in the solver but compute it on the fly whenever you need it

This way you won't risk that someone forgets and accesses sth wrong

@msandre this is at least from what I understood what was your idea initially.

@rubenzorrilla does this make sense to you?

Reply:
For the elements in the FluidDynamicsApplication, this would imply to call the lightweight classes when filling the data structure of each one of the elements in the mesh. This is done for each element at each non-linear iteration, so I guess that the total overhead would be significantly larger. 

Besides, it would imply to hardcode the order of the BDF scheme (now it is hardcoded, however this might be improved in the future) or to use the wrapping BDF I created, which would imply a switch statement in each element at each non-linear iteration.